### DISCUSSION TITLE
Provide for positive quick self-test  exposure notifications (bypassing OTK mechanism)

### DISCUSSION THREAD

**Discussion by LeCanardQuoi** [cds-snc/covid-alert-documentation#39] [Ideas] [answered]
# Summary | Résumé
Self-test kits similar to pregnancy tests are becoming commonly available and the app should better respond to such new tooling.  As a user of the app, I expect to be timely notified if I have been in the proximity of an other app user that took on one of these self-test kits returning positive (and vice-versa). I expect the notice to be disclaimed as such (i.e. not further validated by a provincial/local health authority) and include a reference to the kit name and number. The notice should include critical timestamp info so that I can check my own personal private records (e.g. Google Location Trace and Calendar), my exposure risk then (such as was I wearing a mask, was the location properly aerated, was I “socializing”, etc.) and take further test/isolation action appropriately (e.g. notify own bubble members, put myself in quarantine, etc.). This critical info could become available for contact tracing follow up from the provincial/local health authority (under the control of the app user) . 

# Design detail | Précisions au sujet de la conception
The app user interface should be enhanced with a “Notify others of  quick self-test positive results” and parameters: test kit name, number and result details. Related events would be logged. Proximity contact app users that were tracked during suspected period time frame would be soon notified (with proper disclaimer). The notification should also be sent out to the health portal of the provincial/local health authority (if they have signed up) for further action at their end, including notifying contact tracing folks. A validation of the credentials of the app user could be added at a later date (e.g. number of notices sent out before), in order to prevent rogue users from abusing such a feature (even if notice comes with a disclaimer).

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet
The stronger response plan to enable contact tracing for dealing with new strains remains publicly unknown at this point. The hereby proposed scheme could buy time in better responding to these more viral strains before the OTK process becomes streamlined cds-snc/covid-alert-app#1347. This proposed enhancement request also repurposes the app and lessens the current critical dependency on provincial, regional and territorial health authorities that have not signed on for the app, or too slow to respond in various scenarios. Appropriate messaging/help should support such an enhancement, over and above the disclaimer. Not sure how the Kiwi app and others are managing these disruptive self-test kits? Could certainly be part of an app relaunch plan yet to be publicly documented.

**Comment by obrien-j** [comment]
Transferring over to central repo for greater exposure to new ideas.

**Comment by obrien-j** [comment, marked as answer]
Thanks for the submission @LeCanardQuoi ,

We're working with HC and PHAC (Public Health Agency of Canada) and others on how we approach the rapid innovation in the distributed test space and how we may adjust the app. It's very much an area of focus, as soon as we know more we will update the repo.

**Reply by LeCanardQuoi** [reply]
Definitely the way forward with BC and Alta, in order to decouple these slow moving IT health bureaucracies.  An OTK could in fact be supplied with any positive test results (quick or not) with suitable disclaimer. In combo with more preventative social distancing measures https://github.com/cds-snc/covid-alert-app/issues/1304 , these would really enable contact tracing people (once granted this access to the app user "digital diary" under user control).

**Reply by LeCanardQuoi** [reply]
Area of focus but no progress (or update to repository) in one month?
